common/src/api/external/mod.rs

```diff
@@ -3090,6 +3090,24 @@ pub enum ImportExportPolicy {
     Allow(Vec<oxnet::IpNet>),
 }
 
+/// Query parameters for reading the support bundle
```

Comment:
These are moved from the sled agent API to a common location - they're going to be part of the external Nexus API too.